aten/src/ATen/native/cpu/IndexKernel.cpp

```diff
@@ -526,6 +529,188 @@ void cpu_hflip_vec(at::TensorIterator& iter) {
 }
 
 
+template<int size0>
+void vectorized_cpu_hflip_channels_last_uint8(int64_t & i, char * C10_RESTRICT *data, const int64_t size, const int64_t stride) {
```

Comment:
Taking `i`by reference forces it onto the stack. I would assume iteration starts at 0 and return the value of `i` at the end of the function.